src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -75,14 +80,14 @@ public static void overrideServiceAccountFilePathIfProvided() {
     if (beforeAllFlag) {
       String serviceAccountType = System.getenv("SERVICE_ACCOUNT_TYPE");
       if (serviceAccountType != null && !serviceAccountType.isEmpty()) {
-          if (serviceAccountType.equalsIgnoreCase("FilePath")) {
-            PluginPropertyUtils.addPluginProp("serviceAccountType", "filePath");
-            String serviceAccountFilePath = System.getenv("SERVICE_ACCOUNT_FILE_PATH");
-            if (!(serviceAccountFilePath == null) && !serviceAccountFilePath.equalsIgnoreCase("auto-detect")) {
-              PluginPropertyUtils.addPluginProp("serviceAccount", serviceAccountFilePath);
-            }
-            return;
+        if (serviceAccountType.equalsIgnoreCase("FilePath")) {
+          PluginPropertyUtils.addPluginProp("serviceAccountType", "filePath");
+          String serviceAccountFilePath = System.getenv("SERVICE_ACCOUNT_FILE_PATH");
+          if (!(serviceAccountFilePath == null) && !serviceAccountFilePath.equalsIgnoreCase("auto-detect")) {
+            PluginPropertyUtils.addPluginProp("serviceAccount", serviceAccountFilePath);
           }
+          return;
+        }
```

Comment:
The indentation was changed here but the code changes are unrelated to the PR, please revert the changes so its not highlighted. (Review all the changes for this)